src/doctopus/db.clj

```diff
@@ -0,0 +1,90 @@
+(ns doctopus.db
+  (:require [korma.db :refer [defdb sqlite3]]
+            [clojure.string :as string :refer [split-lines]]
+            [korma.core :refer :all]
+            [clj-time.format :as time-format]
+            [clj-time.core :as clj-time]
+            [doctopus.configuration :refer [server-config]]))
+
+(def standard-datetime (time-format/formatters :basic-date-time))
```

Comment:
ah, good point! and no, i need to update it to give us the datetime in the format we like best